modules/distribution/src/repository/resources/dbscripts/apimgt/oracle.sql

```diff
@@ -734,12 +733,12 @@ CREATE OR REPLACE TRIGGER IDP_LOCAL_CLAIM_TRIG
 /
 CREATE TABLE IDN_ASSOCIATED_ID (
             ID INTEGER,
-        IDP_USER_ID VARCHAR(255) NOT NULL,
+	    IDP_USER_ID VARCHAR(255) NOT NULL,
             TENANT_ID INTEGER DEFAULT -1234,
-        IDP_ID INTEGER NOT NULL,
+	    IDP_ID INTEGER NOT NULL,
             DOMAIN_NAME VARCHAR(255) NOT NULL,
-        USER_NAME VARCHAR(255) NOT NULL,
-        PRIMARY KEY (ID),
+ 	    USER_NAME VARCHAR(255) NOT NULL,
+	    PRIMARY KEY (ID),
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04